packages/plugins/i18n/server/src/graphql.ts

```diff
@@ -47,185 +32,94 @@ export default ({ strapi }: { strapi: Strapi }) => ({
-          ...getLocalizedCreateMutationsResolversConfigs({ typeRegistry }),
-
           // Modify the default scope associated to find and findOne locale queries to match the actual action name
           'Query.i18NLocale': { auth: { scope: 'plugin::i18n.locales.listLocales' } },
           'Query.i18NLocales': { auth: { scope: 'plugin::i18n.locales.listLocales' } },
         },
       };
     });
+  },
+});
 
-    const getLocaleScalar = ({ nexus }: any) => {
-      const locales = getI18NService('iso-locales').getIsoLocales();
-
-      return nexus.scalarType({
-        name: LOCALE_SCALAR_TYPENAME,
-
-        description: 'A string used to identify an i18n locale',
-
-        serialize: identity,
-        parseValue: identity,
-
-        parseLiteral(ast: any) {
-          if (ast.kind !== 'StringValue') {
-            throw new ValidationError('Locale cannot represent non string type');
-          }
-
-          const isValidLocale = ast.value === 'all' || locales.find(propEq('code', ast.value));
-
-          if (!isValidLocale) {
-            throw new ValidationError('Unknown locale supplied');
-          }
-
-          return ast.value;
-        },
-      });
-    };
-
-    const getCreateLocalizationMutations = ({ nexus, typeRegistry }: any) => {
-      const localizedContentTypes = getLocalizedTypesFromRegistry({ strapi, typeRegistry }).map(
-        prop('config.contentType')
-      );
-
-      const createLocalizationComponents = localizedContentTypes.map((ct: any) =>
-        getCreateLocalizationComponents(ct, { nexus })
-      );
+const getLocaleScalar = ({ nexus }: any) => {
+  const { service: getI18NService } = strapi.plugin('i18n');
 
-      // Extract & merge each resolverConfig into a single object
-      const resolversConfig = createLocalizationComponents
-        .map(prop('resolverConfig'))
-        .reduce(merge, {});
+  const locales = getI18NService('iso-locales').getIsoLocales();
 
-      const mutations = createLocalizationComponents.map(prop('mutation'));
+  return nexus.scalarType({
+    name: LOCALE_SCALAR_TYPENAME,
 
-      return { mutations, resolversConfig };
-    };
+    description: 'A string used to identify an i18n locale',
 
-    const getCreateLocalizationComponents = (contentType: any, { nexus }: any) => {
-      const { getEntityResponseName, getContentTypeInputName } = getGraphQLService('utils').naming;
-      const { createCreateLocalizationHandler } = getI18NService('core-api');
+    serialize: identity,
+    parseValue: identity,
 
-      const responseType = getEntityResponseName(contentType);
-      const mutationName = getCreateLocalizationMutationType(contentType);
+    parseLiteral(ast: any) {
+      if (ast.kind !== 'StringValue') {
+        throw new ValidationError('Locale cannot represent non string type');
+      }
 
-      const resolverHandler = createCreateLocalizationHandler(contentType);
+      const isValidLocale = ast.value === 'all' || locales.find(propEq('code', ast.value));
```

Comment:
I'm not sure if I remember whether we're switching to '*' instead of 'all' for locales? Or is it still in discussion?

Reply:
I'll move to * to be consistent with the rest API good catch